orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -350,10 +306,14 @@ def to(self, *args, **kwargs):
     def eval(self: T) -> T:
         self._is_training = False
         self._original_module.eval()
+        if self._flattened_output_module:
+            self._flattened_output_module.eval()
 
     def train(self: T, mode: bool = True) -> T:
         self._is_training = mode
         self._original_module.train(mode)
+        if self._flattened_output_module:
```

Comment:
If `self._flattened_output_module` is initialized from the ctor, you could even remove `self._original_module.train()` call, right?

Reply:
I think so. The only thing is that currently on ```eval```, we call the ```original_module``` instead of ort. So, on eval, I would still need to invoke the original module. Probably, we are better of setting both here.